modules/default/calendar/calendarutils.js

```diff
@@ -417,7 +417,8 @@ const CalendarUtils = {
 
 						endDate = moment(parseInt(startDate.format("x")) + duration, "x");
 						if (startDate.format("x") === endDate.format("x")) {
-							endDate = endDate.endOf("day");
+							// see https://forum.magicmirror.builders/topic/17380/calendar-event-same-start-and-enddate-wrong-google-calendar
```

Comment:
Please describe in the comment why this gets removed, the link to the forum could one day be gone :-) And maybe remove the whole if-block since it isnt used anymore?